examples/hello_world.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use anyhow::Result;
+use proxy_wasm_test_framework::{tester, types::*};
+use std::env;
+
+fn main() -> Result<()> {
+    let args: Vec<String> = env::args().collect();
+    assert_eq!(args.len(), 2);
+    let hello_world_path = &args[1];
+    let mut hello_world_test = tester::test(hello_world_path)?;
+
+    hello_world_test
+        .call_start()
+        .execute_and_expect(ReturnType::None)?;
+
+    let root_context = 1;
```

Comment:
Uhm, is that a recent change? It used to be that the root context was always created with `parent_context_id = 0`, and that's how some SDKs are inferring the type of the context to create (RootContext if `parent_context_id == 0`, {Http,Stream}Context otherwise).